src/xcode/ENA/ENA/Source/Services/PPAccessControl/Model/SRSErrorAlert.swift

```diff
@@ -0,0 +1,47 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+/// The type of SRS error alert, depending on a corresponding error
+enum SRSErrorAlert: String, CaseIterable {
```

Comment:
Theoretically this could als be framed as some kind of `LocalizedError` and be called e.g. `SRSAlertError` that uses the `errorDescription` as the message. But it's also just fine the way it is, just wanted to mention it, I think we used it the other way around in some other places.

Reply:
Thanks for the feedback. I would leave it.